ports/rp2/boards/RPI_PICO2_W/mpconfigvariant_RISCV.cmake

```diff
@@ -0,0 +1 @@
+set(PICO_PLATFORM "rp2350-riscv")
```

Comment:
The RPI_PICO2 board also has a `mpconfigvariant.cmake` file which is used for the default variant (ARM) and contains:
```
set(PICO_PLATFORM "rp2350")
```
It looks like this is not actually needed because the pico-sdk automatically infers the PICO_PLATFORM based on PICO_BOARD (if PICO_PLATFORM is not given).  But maybe it's worth being explicit here and including `mpconfigvariant.cmake` with this line to set the platform to `rp2350`?